app.json

```diff
@@ -1,21 +1,11 @@
 {
-  "name": "docs",
-  "description": "Ably docs",
-  "scripts": {
-  },
-  "env": {
-  },
-  "addons": [
-  ],
-  "buildpacks": [
-    {
-      "url": "https://github.com/heroku/heroku-buildpack-ruby.git"
-    },
-    {
-      "url": "https://github.com/ably-forks/heroku-buildpack-nanoc.git"
-    },
-    {
-      "url": "https://github.com/heroku/heroku-buildpack-static"
-    }
-  ]
+    "buildpacks": [
+        {
+            "url": "heroku/nodejs"
+        },
+        {
+            "url": "https://github.com/heroku/heroku-buildpack-static"
+        }
+    ]
 }
+  
```

Comment:
Strange github flags these newline warnings only in some places

Reply:
Turns out the newlines had whitespace on for these files, unsure of the cause but have deleted from here & eslintrc.json